src/lib/addons/feature-event-formatter-md.ts

```diff
@@ -100,7 +100,7 @@ export class FeatureEventFormatterMd implements FeatureEventFormatter {
         if (type === FEATURE_ARCHIVED) {
             return `${this.unleashUrl}/archive`;
```

Comment:
Possible improvement: If `project` is present, we can provide `${this.unleashUrl}/projects/${project}/archive` instead of `${this.unleashUrl}/archive`